openedx/core/djangoapps/api_admin/tests/test_models.py

```diff
@@ -69,3 +73,41 @@ def test_unicode(self):
             unicode(ApiAccessConfig(enabled=False)),
             u'ApiAccessConfig [enabled=False]'
         )
+
+
+class ApiAccessRequestSignalTests(TestCase):
```

Comment:
You'll need to use `unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls')` here so that the tests don't run under Studio settings -- I think that takes care of all the failing Python tests.